src/ui/components/EditableCard.stories.tsx

```diff
@@ -0,0 +1,50 @@
+import { Meta, StoryObj } from '@storybook/react';
+import { EditableCard } from '@/ui/components/EditableCard';
+
+const meta: Meta<typeof EditableCard> = {
+  /* 👇 The title prop is optional.
+   * See https://storybook.js.org/docs/react/configure/overview#configure-story-loading
+   * to learn how to generate automatic titles
+   */
+  title: 'components/EditableCard',
+  component: EditableCard,
+  args: {
+    iconName: 'angle-down',
+    title: 'Card title',
+    description: 'This is the description',
+    value: 'This is the value',
+    actions: [],
+  },
+};
+
+export default meta;
+type Story = StoryObj<typeof EditableCard>;
+
+export const Primary: Story = {
+  args: {
+    title: 'Card title2',
+  },
+};
+
+export const Profile: Story = {
+  args: {
+    iconName: 'user',
+    title: 'Profile',
+    description: 'Signed in as',
+    value: 'Marty McFly',
+    actions: [{ name: 'Log out…', iconName: 'sign-out-alt', danger: true, type: 'link' }],
```

Comment:
Because we actually expose all button props here, I think it would be better to just pass in a button. It would be an EditableCardButton that takes care of most of the logic we want here.